### ISSUE TITLE
Prisma 2 not working

### ISSUE THREAD

**Issue by LeoMartinDev** [nestjs/nest#6548]
## Current behavior
Prisma 2 is not working with current NestJS.
I'm also having a hard time finding why as I'm not able to make NestJS output logs 🤔 
![image](https://user-images.githubusercontent.com/7238941/109614937-90557400-7b33-11eb-9467-663ab120f63f.png)

## Input Code
Follow recipe about Prisma https://docs.nestjs.com/recipes/prisma#prisma

## Expected behavior
-

## Possible Solution
IDK but might be related directly to Prisma and not NestJS 

## Environment

<pre><code>
Nest version: 7.0.0
Typescript version: 4.1.5

 
For Tooling issues:
- Node version: v14.15.0
- Platform:  WSL 2 Ubuntu

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
Please, report this issue in the Prisma repository.

**Comment by LeoMartinDev** [comment]
FYI https://github.com/prisma/prisma/issues/5938

**Comment by LeoMartinDev** [comment]
**TL;DR** install Typescript `4.3.0-dev.2021030` or `npm i -D typescript@next`
